JYU-LB

```diff
@@ -0,0 +1 @@
+Subproject commit c7d07ba1666966f7c1cf5a8202d078240cf43ab4
```

Comment:
I just made a release 0.1.1 for JYU-LB which points to this commit. Git submodule points exactly to one commit on the remote branch.